### DISCUSSION TITLE
Worn Tape v3

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#177] [Presets]
I do not think I posted my other worn tape presets, but after processing them over and over again, I came out with 3. Here are the evolutions:

<details><summary>Worn v1</summary>
<p>

![image](https://github.com/user-attachments/assets/8cdc83db-005d-494d-a2b9-db0786a999b9)

</p>
</details> 

<details><summary>Worn v2</summary>
<p>

![image](https://github.com/user-attachments/assets/057663fe-c38d-4284-a9b0-61baab4189d6)

</p>
</details> 

<details><summary>Worn v3</summary>
<p>

![image](https://github.com/user-attachments/assets/7eb60910-dfd0-49f7-a7a6-f427646ae4e1)

</p>
</details> 

If you would like any of the older presets, let me know. For now, here is the Worn v3 Preset.

[Worn-Tape3.json](https://github.com/user-attachments/files/16847777/Worn-Tape3.json)

>[!NOTE]
>The sample image was based off of a quick sample video for an NBC News introduction. _The NBC logo is a registered trademark of NBC. All rights reserved._

**Comment by smfthe1199** [comment]


**Reply by Degamisu** [reply]
The v3 was just created. You might have v2.

**Reply by smfthe1199** [reply]
I wanted to update this but thanks. I do have v2 no v3
